src/classes/DataFactoryTest.cls

```diff
@@ -15,5 +15,15 @@ public class DataFactoryTest {
         Account a = (Account) adf.build();
         System.assertEquals(null, a.Id, 'build() should not persist in the database');
     }
-
+   	
+   	@IsTest
+    public static void testDifferentFactories() {
+        AccountDataFactory adf = new AccountDataFactory(Account.SObjectType);
+        OpportunityDataFactory odf = new OpportunityDataFactory(Opportunity.SObjectType);
+        Account a = (Account) adf.getDefaultInstance();
```

Comment:
This variable doesn't seem to have a very insightful name. Please give variables names that suggest their purpose.